### PR TITLE
first implementation of MappingProjectRepository

### PR DESCRIPTION
**Pull request by bartcharbon**

### CODE REVIEW

File: molgenis-data-mapper/src/main/java/org/molgenis/data/meta/AttributeMappingMetaData.java [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 public class AttributeMappingMetaData extends DefaultEntityMetaData
 {
-	public static final String ENTITY_NAME = "attributes";
+	public static final String ENTITY_NAME = "AttributeMapping";
 	public static final String IDENTIFIER = "identifier";
 	public static final String SOURCEATTRIBUTEMETADATA = "sourceAttribureMetaData";
```
Comment (Mark-de-Haan) [inline molgenis-data-mapper/src/main/java/org/molgenis/data/meta/AttributeMappingMetaData.java]: Maybe split these var names with '_' for readability?

Comment (bartcharbon) [inline molgenis-data-mapper/src/main/java/org/molgenis/data/meta/AttributeMappingMetaData.java]: I fancy keeping the name exactly like the java classname of the corresponding Object

File: molgenis-data-mapper/src/main/java/org/molgenis/data/mapping/MappingProject.java [hunk 2]
```diff
@@ -7,8 +7,46 @@
 /**
  * Created by charbonb on 14/01/15.
  */
-public class MappingProject {
-	private String id;
+public class MappingProject
+{
+	private String identifier;
 	private MolgenisUser owner;
 	private List<EntityMapping> entityMappings;
```
Comment (Mark-de-Haan) [inline molgenis-data-mapper/src/main/java/org/molgenis/data/mapping/MappingProject.java]: Doesnt a Mapping project require a target entity?

Comment (Mark-de-Haan) [inline molgenis-data-mapper/src/main/java/org/molgenis/data/mapping/MappingProject.java]: screen shot 2015-01-15 at 14 13 52

**Pull request merged**